apps/xprof_gui/priv/package.json

```diff
@@ -22,6 +22,7 @@
     "bootswatch": "3.3.7",
     "c3": "0.4.11",
     "jquery": "2.1.4",
+    "lodash": "^4.17.4",
```

Comment:
TBH: I'd pin it to exact version - just to be safe. :wink:

Reply:
And that's the reason why `package.json` was reformated - I think you've used `npm install --save`. :wink:

Reply:
Do want to lock only version of xprof's dependencies or also dependencies used by packages? Sticking to the exact version in package.json don't cover as from discrepancies. if we want so we should include package-lock.json in the repo. With it we will have consistent <b>node_module</b> tree. Exact versioning in package.json and use of package-lock.json will prevent us from that (npm >= 5 required)

Reply:
including package-lock file sounds like a good idea. we also have rebar.lock for erlang dependencies already commited.

Reply:
including `package-lock` sounds like a good idea. `rebar.lock` for erlang dependencies is already committed.